scalableminds/webknossos
# Add annotation backlink to export job

… so that the job list can link directly to the annotation the job was started from.

![image](https://user-images.githubusercontent.com/1390035/114001282-131ccd80-985c-11eb-8c00-363482e86033.png)


### Steps to test:
- set jobsEnabled = true, set up https://github.com/scalableminds/webknossos-worker/pull/14
- create volume/hybrid annotation, export a bbox
- job list should contain backlink
- old jobs should also still be rendered (without the backlink)

### Issues:
- fixes #5261 

------
- [x] Needs wk-worker update after deployment
